sdk/terraform/Azure.ResourceManager.Terraform/tests/Tests/ExportTerraformTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Terraform.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Terraform.Tests.Tests
+{
+    [TestFixture]
```

Comment:
please remove this attribute - the test framework handles everything and in my local test, this may have some issues.

Reply:
Removed in f86ae1e58ee